vrk-kpa/suomidigi
# SUOPA-654 Blog migration

To test: 
- Copy the XML file to `/public/sites/default/files/`
- `make fresh shell`
- `drush migrate-import suomidigi_wordpress_images --feedback="50 items" && drush migrate-import suomidigi_wordpress_media --feedback="50 items" && drush migrate-import suomidigi_wordpress_tags --feedback="50 items" && drush migrate-import suomidigi_wordpress_paragraphs --feedback="50 items" && drush migrate-import suomidigi_wordpress_content_post --feedback="50 items"`
- Login to the site and go to https://suomidigi.docker.sh/admin/content?type=blog_post
   - Open few of the blog posts to new tabs and check that the content has been migrated (tags, dates, title and content)
   - All of the posts will be curated later on, so no need to worry about missing images, etc.
